File: src/main/java/me/miki/shindo/management/quickplay/impl/UHCQuickPlay.java
package me.miki.shindo.management.quickplay.impl;

import me.miki.shindo.management.quickplay.QuickPlay;
import me.miki.shindo.management.quickplay.QuickPlayCommand;
import net.minecraft.util.ResourceLocation;

import java.util.ArrayList;

public class UHCQuickPlay extends QuickPlay {

    public UHCQuickPlay() {
        super("UHC", new ResourceLocation("shindo/icons/hypixel/UHC.png"));
    }

    @Override
    public void addCommands() {
        ArrayList<QuickPlayCommand> commands = new ArrayList<QuickPlayCommand>();

        commands.add(new QuickPlayCommand("Lobby", "/l hc"));
        commands.add(new QuickPlayCommand("Solo", "/play uhc_solo"));
        commands.add(new QuickPlayCommand("Teams", "/play uhc_teams"));
        commands.add(new QuickPlayCommand("Events Mode", "/play /play uhc_events"));

        this.setCommands(commands);
    }
}

File: src/main/kotlin/me/miki/shindo/management/quickplay/impl/UHCQuickPlay.kt
package me.miki.shindo.management.quickplay.impl

import me.miki.shindo.management.quickplay.QuickPlay
import me.miki.shindo.management.quickplay.QuickPlayCommand
import net.minecraft.util.ResourceLocation

class UHCQuickPlay : QuickPlay("UHC", ResourceLocation("shindo/icons/hypixel/UHC.png")) {

    override fun addCommands() {
        setCommands(
            arrayListOf(
                QuickPlayCommand("Lobby", "/l hc"),
                QuickPlayCommand("Solo", "/play uhc_solo"),
                QuickPlayCommand("Teams", "/play uhc_teams"),
                QuickPlayCommand("Events Mode", "/play uhc_events")
            )
        )
    }
}
